app/vmalert/static/js/custom.js

```diff
@@ -1,38 +1,32 @@
-function expandAll() {
-    $('.group-heading').each(function () {
-        let style = $(this).attr("style")
-        // display only elements that are currently visible
-        if (style === "display: none;") {
-            return
+function actionAll(isCollapse) {
```

Comment:
Click on alert's `Details` or `Source` doesn't work anymore:
<img width="763" alt="image" src="https://github.com/user-attachments/assets/11f76be3-2c8c-4287-aa10-bbddb6d7d473" />

Reply:
fixed it